### ISSUE TITLE
Bad variable name

### ISSUE THREAD

**Issue by ironpowertga** [bcl/aisparser#39]
For c++ dev : 
  int true;              //!&lt; 9 bits  : True Heading
this name is very bad. true is defined in c++. 
trueHeading must be better

char virtual; //!< 1 bit    : Virtual/Pseudo AtoN Flag
same problem with virtual, it's c++ keyword

**Comment by smithcv** [comment]
yeah I just renamed them for my project.

**Comment by bcl** [comment]
Yeah, given the age of this code and that it isn't really C++ that's just how it's going to have to be.
